settings.gradle

```diff
@@ -1,2 +1,3 @@
 rootProject.name = "ProxyToggle"
 include ':app'
+include ':feature-proxy-manager'
```

Comment:
Second include not needed and we can comma separate?

Reply:
👍